mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -1378,7 +1379,8 @@ def run_task(self):  # {{{
         self.logger.info('   Plot time series of max Atlantic MOC at 26.5N...')
         xLabel = 'Time [years]'
         yLabel = '[Sv]'
-        title = r'Max Atlantic MOC at $26.5\degree$N\n {}'.format(mainRunName)
+        title = '{}\n{}'.format(r'Max Atlantic MOC at $26.5\degree$N',
+                                mainRunName)
```

Comment:
The `r` indicates that the characters are taken literally, meaning you can't have escaped characters like `\n`.  You could escape all the characters instead of using `r` (e.g. `\$` and `\\degree`) or you can use the literal string (with `r`) as one of the format arguments as I have done.

Reply:
great. No wonder I wasn't able to fix it easily before..